wiki-config

```diff
@@ -1,16 +1,38 @@
-# wiki-config
+#!/bin/bash
+
+#set -x
+set -e
+set -o pipefail
+set -o nounset
+
+## TODO:
+## create a file:
+## credentials
+## with the following contents:
+#[[ -v USERNAME ]] || USERNAME="username"
+#[[ -v USERPASS ]] || USERPASS="password"
 
 # Location of wiki's api.php
-WIKIAPI="https://wiki.example/Wiki/api.php"
+[[ -v WIKIAPI ]] || WIKIAPI="https://www.whonix.org/w/api.php"
 
 # Location of wiki's index.php
-WIKIINDEX="https://wiki.example/Wiki/index.php"
+[[ -v WIKIINDEX ]] || WIKIINDEX="https://www.whonix.org/w/index.php"
 
-# Username to log in as
-USERNAME="myuser"
+[[ -v TMPFOLDER ]] || TMPFOLDER=/tmp/kicksecure-auto-deploy
+mkdir -p "$TMPFOLDER"
```

Comment:
don't hard code temporary directories in scripts (and that name has nothing to do with this script anyway).
best is to not use temporary directories, which the script didn't before.
if they are _really_ needed, use `mktemp`.
when you make a temporary directory, ensure there is a trap to remove them on exit.